tests/utils/getScenarioContent.test.tsx

```diff
@@ -0,0 +1,83 @@
+import getScenarioContent, { getScenario, ScenarioType } from '../../utils/getScenarioContent'
+import { ScenarioContent } from '../../types/common'
+
+// Shared test constants
+const pendingDeterminationScenario = { pendingDetermination: ['temporary text'] }
+const basePendingScenario = { hasPendingWeeks: true }
+const baseNoPendingScenario = { hasPendingWeeks: false }
+
+/**
+ * Begin tests
+ */
+
+// Test getScenarioContent()
+describe('Retrieving the scenario content', () => {
+  it('returns the correct status description for the scenario', () => {
+    const pendingDetermination: ScenarioContent = getScenarioContent(pendingDeterminationScenario)
+    expect(pendingDetermination.statusContent.statusDescription).toBe('claim-status:pending-determination.description')
+
+    const basePending: ScenarioContent = getScenarioContent(basePendingScenario)
+    expect(basePending.statusContent.statusDescription).toBe('claim-status:base-pending.description')
+
+    const baseNoPending: ScenarioContent = getScenarioContent(baseNoPendingScenario)
+    expect(baseNoPending.statusContent.statusDescription).toBe('claim-status:base-no-pending.description')
+  })
+})
+
+// Test getScenario()
+describe('Identifying the scenario', () => {
```

Comment:
Could we add groupings based on the expected results? Clustering like that will help me parse all the tests more cleanly